component/tests/golden/vshn/appcat/appcat/21_composition_objectstorage_minio.yaml

```diff
@@ -63,6 +64,7 @@ spec:
         apiVersion: v1
         data:
           providerConfig: minio-cluster
+          proxyEndpoint: host.docker.internal:9443
```

Comment:
why this proxyEndpoint?

Reply:
ah whops, forgot to disable it before generating the goldens.